accounts/admin.py

```diff
@@ -0,0 +1,15 @@
+from django.contrib import admin
+from .models import Profile
+
+
+# Register your models here.
+class ProfileAdmin(admin.ModelAdmin):
+    # Defining fields for Admin panel
+    fields = (
+        'slack_display_name',
+        'user_type',
+        'current_lms_module',
+    )
+
+
+admin.site.register(Profile, ProfileAdmin)
```

Comment:
As per PEP8 please add a trailing blank line. Same goes for all `.py` files below please.

Reply:
Now that one is an odd one....I have trailing blank lines on this file explicitely for example. Any .py file that was manually created or developed upon has those blank trailing lines as per PEP8, my PyCharm screams if I don't include them so they are very second nature to be included now. See image below.....the file name being `white` means that it has been commited as such.

![blankLine](https://user-images.githubusercontent.com/48055438/96579621-0a950280-12cf-11eb-840d-60ca9a42e390.PNG)

Is this something that Github does to remove them for example?

Reply:
It seems Github removes these trailing blank lines. I added extra lines to check and they were gone in the actual file themselves in the branch on the repo. See hacktoberfest channel in slack for more details.